vmm/sandbox/src/stratovirt/hooks.rs

```diff
@@ -47,4 +52,11 @@ impl Hooks<StratoVirtVM> for StratoVirtHooks {
         sandbox.sync_clock().await;
         Ok(())
     }
+
+    async fn post_stop(&self, _sandbox: &mut KuasarSandbox<StratoVirtVM>) -> Result<()> {
```

Comment:
`post_stop` was called every time before the sandbox stopped, so it's not appropriate call it here. 

To make it easy, how about not positive stopping tracing, stopping it with the whole process exits?